portal-impl/src/com/liferay/portlet/documentlibrary/service/http/DLFolderServiceHttp.java

```diff
@@ -95,13 +95,58 @@ public static com.liferay.document.library.kernel.model.DLFolder addFolder(
 		}
 	}
 
+	public static com.liferay.document.library.kernel.model.DLFolder addFolder(
```

Comment:
Is this generated code? In that case we should put it in a commit with generated code